client/components/web-preview/component.jsx

```diff
@@ -63,6 +63,8 @@ export class WebPreviewModal extends Component {
 		fixedViewportWidth: PropTypes.number,
 		// Prevents tabbing into the iframe.
 		disableTabbing: PropTypes.bool,
+		// Allow specific functionality for Launchpad
+		isLaunchpad: PropTypes.bool,
```

Comment:
I see we're adding the isLaunchpad prop to ensure changes in this PR only apply to Launchpad. 

Should we also set the default to false in the defaultProps below?